docs/openhabian.md

```diff
@@ -331,7 +331,7 @@ If the installation was **not successful** you will see a warning and further in
 ##### What Next?
 
 After your first setup is done, you might want to generate [Items](http://docs.openhab.org/configuration/items.html), [Sitemap](http://docs.openhab.org/configuration/sitemaps.html) and [HABPanel](http://docs.openhab.org/addons/uis/habpanel/readme.html) dashboard for your home.
-Simply log in to your Raspberry PI with SSH and type `openhab-generator`. It will run a Yeoman-based [openHAB Generator](https://github.com/kubawolanin/generator-openhab) tool asking for your house structure (e.g. number of floors, rooms and devices).
+Simply checkout the generator which could be found at the openHAB dashboard.
```

Comment:
I would mention [Home Builder](https://docs.openhab.org/configuration/homebuilder.html) here as it's a natural evolution of `openhab-generator` in form of UI add-on.